packages/transformers/sass/src/SassTransformer.js

```diff
@@ -0,0 +1,76 @@
+// @flow strict-local
+
+import {Transformer} from '@parcel/plugin';
+import localRequire from '@parcel/local-require';
+import {promisify, parseCSSImport} from '@parcel/utils';
+
+export default new Transformer({
+  async getConfig({asset, resolve}) {
+    let config = await asset.getConfig(['.sassrc', '.sassrc.js'], {
+      packageKey: 'sass'
+    });
+
+    if (config === null) {
+      config = {};
+    }
+
+    config.file = asset.filePath;
+
+    if (config.importer === undefined) {
+      config.importer = [];
+    } else if (!Array.isArray(config.importer)) {
+      config.importer = [config.importer];
+    }
+
+    config.importer = [
+      ...config.importer,
+      resolvePathImporter({asset, resolve})
+    ];
+
+    return config;
+  },
+
+  async transform({asset, config}) {
+    let sass;
+    // Use Node Sass if installed, but prefer Dart Sass.
+    try {
+      sass = await localRequire('node-sass', asset.filePath, true);
```

Comment:
We should probably drop support for `node-sass` it caused issues in Parcel 1

Reply:
Let's warn about `node-sass` not being supported by Parcel if it's installed then.